### DISCUSSION TITLE
Kibana 404

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#12712] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200G

### Storage for /nsm

320G

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

When going to kibana I am getting 404 error.  Kibana service is running.  The only service that is having issues is elastalert.  Below is what I am seeing in my elasticsearch logs.

```
[2024-04-01T13:44:00,023][ERROR][org.elasticsearch.xpack.security.action.filter.SecurityActionFilter] blocking [cluster:monitor/nodes/stats] operation due to expired license. Cluster health, cluster stats and indices stats
operations are blocked on license expiration. All data operations (read and write) continue to work.
If you have a new license, please update it. Otherwise, please reach out to your support contact.
[2024-04-01T13:44:19,456][ERROR][org.elasticsearch.xpack.security.action.filter.SecurityActionFilter] blocking [cluster:monitor/nodes/stats] operation due to expired license. Cluster health, cluster stats and indices stats
operations are blocked on license expiration. All data operations (read and write) continue to work.
If you have a new license, please update it. Otherwise, please reach out to your support contact.
[2024-04-01T13:44:19,456][WARN ][org.elasticsearch.cluster.InternalClusterInfoService] failed to retrieve node stats
org.elasticsearch.ElasticsearchSecurityException: current license is non-compliant for [security]
```
From running salt-call state.highstate I am getting this.
```
local:
    Data failed to compile:
----------
    The function "state.highstate" is running as PID 165866 and was started at 2024, Apr 01 13:45:59.201650 with jid 20240401134559201650
```
From kibana logs I am seeing this.
`{"service":{"node":{"roles":["background_tasks","ui"]}},"ecs":{"version":"8.6.1"},"@timestamp":"2024-04-01T13:28:09.174+00:00","message":"[.kibana_security_solution] Action failed with 'security_exception\n\tRoot causes:\n\t\tsecurity_exception: current license is non-compliant for [security]'. Retrying attempt 15 in 64 seconds.","log":{"level":"ERROR","logger":"savedobjects-service"},"process":{"pid":7},"trace":{"id":"5082275cfc282cd9e23d5e1ad671af3f"},"transaction":{"id":"1c721f099ea59f37"}}`

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Did you have an Elastic license? What does `sudo so-elasticsearch-query _license ` return?

**Comment by intsec** [comment]
Status shows expired, type: trial issued_to securityonion

**Reply by cm-ops** [reply]
Do you have the option in `Kibana - Stack Management - License Management - Revert to Basic`? You can also use the API to do this https://www.elastic.co/guide/en/elasticsearch/reference/current/start-basic.html#start-basic

**Reply by intsec** [reply]
Well I am getting a 404 when I try to go to Kibana so I tried doing a curl command from the box itself using "curl -X POST "localhost:9200/_license/start_basic?pretty"" and I get this.
`curl: (52) Empty reply from server`

**Reply by cm-ops** [reply]
Try `sudo so-elasticsearch-query _license/start_basic?acknowledge=true -XPOST`.
